client/coral-embed/src/index.js

```diff
@@ -190,7 +190,10 @@ Talk.render = function(el, opts) {
   // Compose the query to send down to the Talk API so it knows what to load.
   let query = {};
 
-  query.comment_id = window.location.hash.slice(1);
+  let urlParams = new URLSearchParams(window.location.search);
```

Comment:
Support for `URLSearchParams` is not good, see: http://caniuse.com/#feat=urlsearchparams.
Needs alternative or polyfill.